.travis.yml

```diff
@@ -2,8 +2,8 @@ sudo: false
 
 language: elixir
 
-elixir: "1.9"
-opt_release: "22.0"
+elixir: "1.11"
```

Comment:
First of all - thanks a lot for the PR!

The workflow is failing because you have bumped the elixir version - do we use something from 1.11? If not I think we should allow running on 1.9 as well (same for opt 22).

Reply:
@suranyami

Reply:
Ah sorry - this comment applies to the mix.exs file 
As far as I can see the workflow file used is this: 
https://github.com/jakobht/azurex/blob/0c5980f5b555dfec09fb35fffed0c4bfde66cb76/.github/workflows/elixir.yml
So if we really want to bump the version we should change that one - but again, I don't really see a reason to do that. What do you think @Zinoex ?